plugins/modules/monitoring/alerta_customer.py

```diff
@@ -0,0 +1,199 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2022, Christian Wollinger <@cwollinger>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: alerta_customer
+short_description: Manage customers in Alerta
+version_added: 4.8.0
+description:
+  - Create or delete customers in Alerta with the REST API.
+author: Christian Wollinger (@cwollinger)
+seealso:
+  - name: API documentation
+    description: Documentation for Alerta API
+    link: https://docs.alerta.io/api/reference.html#customers
+options:
+  customer:
+    description:
+      - Name of the customer.
+    required: true
+    type: str
+  match:
+    description:
+      - The matching logged in user for the customer.
+    required: true
+    type: str
+  alerta_url:
+    description:
+      - The Alerta API endpoint.
+    required: true
+    type: str
+  api_username:
+    description:
+      - The username for the API using basic auth.
+    type: str
+  api_password:
+    description:
+      - The password for the API using basic auth.
+    type: str
+  api_key:
+    description:
+      - The access token for the API.
+    type: str
+  state:
+    description:
+      - Whether the customer should exist or not.
```

Comment:
How about specifying here what exactly is created or deleted:
```suggestion
      - Whether the customer should exist or not.
      - Both I(customer) and I(match) identify a customer that should be added or removed.
```

Reply:
changed